src/main/resources/lib/credentials/select/select.js

```diff
@@ -260,6 +260,17 @@ Behaviour.specify("INPUT.credentials-select", 'credentials-select', -100, functi
   }).bind($(x));
   x.onchange();
 });
+Behaviour.specify("DIV.include-user-credentials", 'include-user-credentials', 0, function (e) {
+    e.querySelector("input[name='includeUser']").addEventListener('click', function (evt) {
+        var caution = e.querySelector("span.user-credentials-caution");
+        if (this.checked) {
+            caution.setAttribute('style', 'display:inline');
+        } else {
+            caution.setAttribute('style', 'display:none');
+        }
+        layoutUpdateCallback.call();
```

Comment:
Now that the caution message is displayed inline, this might not be needed.

Reply:
Please remove it if it's not needed :)